contrib/opencensus-ext-stackdriver/opencensus/ext/stackdriver/stats_exporter/__init__.py

```diff
@@ -65,70 +67,67 @@
 
 
 class Options(object):
-    """ Options contains options for configuring the exporter.
+    """Exporter configuration options.
+
+     `resource` is an optional field that represents the Stackdriver monitored
+     resource type. If unset, this defaults to a `MonitoredResource` with type
+     "global" and no resource labels.
+
+     `default_monitoring_labels` are labels added to every metric created by
+     this exporter. If unset, this defaults to a single label with key
+     "opencensus_task" and value "py-<pid>@<hostname>". This default ensures
+     that the set of labels together with the default resource (global) are
+     unique to this process, as required by stackdriver.
+
+     If you set `default_monitoring_labels`, make sure that the `resource`
+     field together with these labels is unique to the current process. This is
+     to ensure that there is only a single writer to each time series in
+     Stackdriver.
+
+     Set `default_monitoring_labels` to `{}` to avoid getting the default
+     "opencensus_task" label. You should only do this if you know that
+     `resource` uniquely identifies this process.
+
+    :type project_id: str
+    :param project_id: The ID GCP project to export metrics to, fall back to
+        default application credentials if unset.
+
+    :type resource: str
```

Comment:
Type should be `google.cloud.monitoring_v3.types.MonitoredResource`.

Reply:
I thought so too, but it actually expects a string in `set_monitored_resource`. This is the monitored resource's (string-valued) type, not the resource itself.

That said, I think it would make sense to change this (and `set_monitored_resource`) to take a `MonitoredResource` instead.

Reply:
Agree, that makes more sense and is consistent with Java and Go options.